src/v2/guide/components-slots.md

```diff
@@ -85,145 +163,396 @@ Para estos casos, el elemento `<slot>` tiene un atributo especial, `name`, el cu
+
+``` html
+<current-user v-slot="{ user: person }">
+  {{ person.firstName }}
+</current-user>
+```
+
+Incluso puede definir las alternativas, para ser usadas en caso de que un atributo del slot no esté definido:
+
+``` html
+<current-user v-slot="{ user = { firstName: 'Guest' } }">
+  {{ user.firstName }}
+</current-user>
+```
+## Nombre dinámicos de los slots
+
+> Nuevo en 2.6.0+
+
+[Argumentos dinámicos de la directiva](syntax.html#Dynamic-Arguments) también funcionan en `v-slot`, permitiendo la definición de nombres dinámicos en los slots:
+
+``` html
+<base-layout>
+  <template v-slot:[dynamicSlotName]>
+    ...
+  </template>
+</base-layout>
+```
+## Nombre abreviados de slots
+
+> Nuevo en 2.6.0+
+
+Similar a `v-on` y `v-bind`, `v-slot` también tiene una abreviatura, reemplazando todo antes del argumento (`v-slot:`) con el símbolo especial "#". Por ejemplo, `v-slot:header` puede ser reescrito como `#header`:
+
+```html
+<base-layout>
+  <template #header>
+    <h1>Aquí podría ir un título de página</h1>
+  </template>
+
+  <p>Un párrafo para el contenido principal.</p>
+  <p>Y otro más.</p>
+
+  <template #footer>
+    <p>Aquí va alguna información de contacto</p>
+  </template>
+</base-layout>
+```
+
+Sin embargo, al igual que con otras directivas, la abreviatura sólo está disponible cuando se proporciona un argumento. Eso significa que la siguiente sintaxis es inválida:
+
+``` html
+<!-- Esto activará una advertencia -->
+<current-user #="{ user }">
+  {{ user.firstName }}
+</current-user>
+```
+
+En cambio, siempre debe especificar el nombre del slot si desea utilizar la abreviatura:
+
+``` html
+<current-user #default="{ user }">
+  {{ user.firstName }}
+</current-user>
+```
+
+## Otros ejemplos
+
+**Las propiedades de slot nos permiten convertir los slots en plantillas reutilizables que pueden mostrar diferentes contenidos basados en las propiedades de los inputs.** Esto es muy útil cuando se diseña un componente reutilizable que encapsula la lógica de los datos mientras permite al componente padre consumidor personalizar parte de su diseño.
+
+Por ejemplo, estamos implementando un componente `<todo-list>` que contiene la lógica de diseño y filtrado para una lista:
 
 ```html
 <ul>
   <li
-    v-for="todo in todos"
+    v-for="todo in filteredTodos"
     v-bind:key="todo.id"
   >
     {{ todo.text }}
   </li>
 </ul>
 ```
 
-Pero en algunas partes de nuestra aplicación, queremos que los elementos individuales de la lista de `todo` realicen algo diferente al `todo.text`. Aquí es donde los slots con scope entran en juego.
-
-Para hacer que la funcionalidad sea posible, todo lo que tenemos que hacer es envolver el contenido del elemento `todo` en un elemento `<slot>`, luego pasar al slot cualquier dato relevante a su contexto: en este caso, el objeto `todo`:
+En lugar de hard-code cada "todo", podemos dejar que el componente padre tome el control haciendo de cada "todo" un slot, y luego vincular `todo` como una propiedad del slot:
```

Comment:
Same here, italic for english words:
```suggestion
En lugar de _hard-code_ cada _todo_, podemos dejar que el componente padre tome el control haciendo de cada _todo_ un slot, y luego vincular `todo` como una propiedad del slot:
```
Also, please change `hard-code` to how it's spelled on the english version (if it's not spelled in this way)